src/mtcp/mtcp_sys.h

```diff
@@ -326,6 +326,7 @@ struct linux_dirent {
                               5,                                          \
                               args)
 # define mtcp_sys_munmap(args ...)    mtcp_inline_syscall(munmap, 2, args)
+# define mtcp_sys_msync(args ...)    mtcp_inline_syscall(munmap, 3, args)
```

Comment:
Isn't it  the ```mtcp_inline_syscall(munmap, 3, args) ``` should be ```mtcp_inline_syscall(msync, 3, args) ``` ?

Reply:
Fixed in my next update to this PR.  Thanks.